README.md

```diff
@@ -59,8 +79,13 @@ const slides = [
 ];
 
 export default class App extends React.Component {
-  this.state = {
-    showRealApp: false
+
```

Comment:
Adding a constructor function is actually not necessary for initializing state, so this change can be removed.